fjord/settings/base.py

```diff
@@ -9,13 +9,8 @@
 
 from fjord.settings_utils import config
 
-
-ROOT = os.path.abspath(
-    os.path.join(
-        os.path.dirname(__file__),
-        '..',
-        '..'
-    ))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+ROOT = os.path.dirname(BASE_DIR)
```

Comment:
`BASE_DIR` ends up on my machine as `/home/willkg/mozilla/fjord/fjord` -- the directory with the fjord apps in it.

`ROOT` ends up on my machine as `/home/willkg/mozilla/fjord` -- the git repository root.